src/components/home/modals/Send.vue

```diff
@@ -626,12 +637,67 @@ export default {
             }, respErr => {
             })
         },
+        recordTmpRecipient(recipient) {
+            let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+            if (recipient.length < 9 && alias) {
+                for (let key in alias) {
+                    if (recipient.toLowerCase() === alias[key].toLowerCase()) {
+                        this.tmpRecipient = key
+                        break
+                    }
+                }
+            } else {
+                this.tmpRecipient = recipient
+            }
+        },
         inputAmount(num) {
             return BigNumber(num)
         },
+        showRecipient() {
+            if (JSON.parse(window.localStorage.getItem(this.defaultAddress)) != null) {
+                let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+                if (this.recipient.length < 9 && alias) {
```

Comment:
Use `this.recipient.length <= MAX_ALIAS_LENGTH` instead of "this.recipient.length < 9". A lot of place used alias length. Save `MAX_ALIAS_LENGTH` to `constants.js` and change all such statements to that.